main.tf

```diff
@@ -43,3 +43,100 @@ module "vpc" {
 
 
 ##############################################################################
+
+
+##############################################################################
+# Create CBR prewired rules for VPC -> COS (scoped to resource group)
+##############################################################################
+module "slz_cbr_zone_vpcs" {
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-zone-module"
+  version          = "1.18.0"
+  name             = "${var.prefix}-slz-vpcs-zone"
+  zone_description = "Single zone grouping all SLZ VPCs."
+  account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+  addresses = [
+    for network in module.vpc :
+    { "type" = "vpc", value = network.vpc_crn }
+  ]
+}
+
+locals {
+
+  cos_data_merge_maps = merge(local.cos_data_map, local.cos_map)
+
+  cos_resource_group_names = distinct(flatten([
+    for key, value in local.cos_data_merge_maps : [
+      value.resource_group
+    ]
+  ]))
+
+  rule_contexts = [{
+    attributes = [
+      {
+        "name" : "endpointType",
+        "value" : "private"
+      },
+      {
+        name  = "networkZoneId"
+        value = module.slz_cbr_zone_vpcs["zone_id"]
+    }]
```

Comment:
Do we need to allow access from the Schematics service here so that it doesn't break deployment? For example, like what we did [here](https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/4755cba6aebe378b3ce145d52266f2e193edeefe/examples/fscloud/main.tf#L86C1-L101C2) and [here](https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/4755cba6aebe378b3ce145d52266f2e193edeefe/examples/fscloud/main.tf#L134-L142)